test/ffi007/ffi007.c

```diff
@@ -0,0 +1,21 @@
+#include <stdio.h>
+
+typedef int (*callback)(int);
+typedef int (*cb2)(char *);
+
+void test_ffi(callback cb)
```

Comment:
Lets name that one `test_ffi1` for consistency with the other ffi test functions.